runtime/kusama/src/lib.rs

```diff
@@ -1258,27 +1258,7 @@ parameter_types! {
 	pub const ThawThrottle: (Perquintill, BlockNumber) = (Perquintill::from_percent(25), 5);
 	pub storage NisTarget: Perquintill = Perquintill::zero();
 	pub const NisPalletId: PalletId = PalletId(*b"py/nis  ");
-	pub const NisHoldReason: HoldReason = HoldReason::Nis(HoldReasonNis::NftReceipt);
-}
-
-/// A reason for placing a hold on funds.
-#[derive(
-	Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, MaxEncodedLen, Debug, TypeInfo,
-)]
-pub enum HoldReason {
-	/// Some reason of the NIS pallet.
-	#[codec(index = 38)]
-	Nis(HoldReasonNis),
-}
-
-/// A reason for the NIS pallet placing a hold on funds.
-#[derive(
-	Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, MaxEncodedLen, Debug, TypeInfo,
-)]
-pub enum HoldReasonNis {
-	/// The NIS Pallet has reserved it for a non-fungible receipt.
-	#[codec(index = 0)]
-	NftReceipt = 0,
+	pub const NisHoldReason: RuntimeHoldReason = RuntimeHoldReason::Nis(pallet_nis::HoldReason::NftReceipt);
```

Comment:
We only need this for tests or?
So can we move it to the test?

Reply:
It's to make sure that the encoding didn't change before and after the code changes in this PR. In the future, it would probably be good to keep this around so that we don't accidentally change the encoding via modifying the `composite_enum` attribute.

Reply:
Ah sorry, I misread. We do need this to configure the `HoldReason` on the NIS pallet.